src/environmentd/tests/sql.rs

```diff
@@ -3875,12 +3875,14 @@ async fn test_explain_timestamp_blocking() {
 
 #[mz_ore::test(tokio::test(flavor = "multi_thread", worker_threads = 1))]
 #[cfg_attr(miri, ignore)] // too slow
-async fn test_linearize_read_then_write() {
```

Comment:
This test was combined with `test_cancel_linearize_reads`. Previously, this test would start a query that never completes. `ClientTransmitter` has a panic in it's drop implementation if a response was never sent to the client. Previously the `ClientTransmitter` lived in an async task that would eat the panic and the test would still pass. Now the `ClientTransmitter` transmitter is stored as a field in the `Coordinator`, so dropping the `Coordinator` also drops the `ClientTransmitter`, which panics and fails the test.